.github/workflows/publish_to_wiki.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish to Wiki
+
+# Trigger this action only if there are changes pushed to the docs/** directory under the main branch
+on:
+  push:
+    paths:
+      - markdown/wiki/** # This includes all sub folders
+    branches:
+      - main # This can be changed to any branch of your preference
+
+env:
+  USER_TOKEN: ${{ secrets.WIKI_ACTION_TOKEN }} # This is the repository secret
+  USER_NAME: <USERNAME> # Enter the username of your (bot) account
+  USER_EMAIL: <EMAIL> # Enter the e-mail of your (bot) account
+  OWNER: ${{ github.event.repository.owner.name }} # This is the repository owner
+  REPOSITORY_NAME: ${{ github.event.repository.name }} # This is the repository name
+
+jobs:
+  publish_to_wiki:
+    name: Publish to Wiki
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install podman
+        run: |
+          apt-get -qq -y podman
```

Comment:
My understanding is these canned ubuntu images already include podman, buildah, and skopeo.